dateparser/utils.py

```diff
@@ -86,3 +86,24 @@ def find_date_separator(format):
     m = re.search(r'(?:(?:%[dbBmaA])(\W))+', format)
     if m:
         return m.group(1)
+
+
+class Registry(type):
```

Comment:
Thank you for the tip! made the changes accordingly. Please let me know how you like the changes.

Regarding users wanting to use their own metaclasses, I am just curious why would they want to do this, of course, we can't be sure of their wants/needs. But provided the scope of our library, we expose certain methods and classes to users and quite often make assumptions that a component won't be required to be used individually. Likewise, our guarantee is only limited and do not cover not-prescribed usage of unexposed/internal components of the library.